src/plugins/embeddable/public/lib/embeddables/error_embeddable.tsx

```diff
@@ -11,6 +11,7 @@ import { EmbeddablePanelError } from '../panel/embeddable_panel_error';
 import { Embeddable } from './embeddable';
 import { EmbeddableInput, EmbeddableOutput, IEmbeddable } from './i_embeddable';
 import { IContainer } from '../containers';
+import './error_embeddable.scss';
```

Comment:
The old `error_embedabble.scss` wasn't actually being used anywhere - now it is 👍

Reply:
Have we checked if this changes anything in the way that the normal error embeddables on a dashboard render?

Reply:
All the rules in the `.scss` file target the specific `errorEmbeddableCompact` classes so no, it shouldn't - just to be sure though, I commented out this import and ensured that this made no impact to the look of error embeddables :+1: 

![image](https://user-images.githubusercontent.com/8698078/217031760-b9741e3e-2754-45e8-9294-763e3a72ef1d.png)